src/types/state.js

```diff
@@ -217,9 +217,10 @@ export type FullProfileSpecificUrlState = {|
 /**
  * Active tab profile specific url state
  * They should not be used from the full view.
- * NOTE: This state is empty for now, but will be used later, do not remove.
  */
-export type ActiveTabSpecificProfileUrlState = {||};
+export type ActiveTabSpecificProfileUrlState = {|
+  resourcesOpen: boolean,
```

Comment:
For this naming, I wasn't sure which one is more correct. I could also do `resourcesPanelOpen(ed)` I think? What do you think? I removed the `Panel` from this because I wanted to keep it as short as possible for the url. But we can use longer on the state and shorter on the url too..

Reply:
Optional: Yeah, generally be as verbose as needed in the UrlState, and then terse in the URL serialization.

It's often nice in boolean variables to start them with `is`, such as `isResourcesPanelOpen`.